### PR TITLE
feat: add discord.js-utilities pkg

### PR DESCRIPTION
**Pull request by RealShadowNova**

### CODE REVIEW

File: packages/discord.js-utilities/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+[![Coverage Status](https://coveralls.io/repos/github/sapphire-project/utilities/badge.svg?branch=main)](https://coveralls.io/github/sapphire-project/utilities?branch=main)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/discord.js-utilities?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/discord.js-utilities)
+[![npm](https://img.shields.io/npm/v/@sapphire/discord.js-utilities?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/discord.js-utilities)
+[![Depfu](https://badges.depfu.com/badges/ec42ff3d6bae55eee1de4749960852b3/count.svg)](https://depfu.com/github/sapphire-project/utilities?project_id=15195)
+
+</div>
+
+## Description
+
+When developing Discord bots or frameworks for Discord bots we often run into certain constants such as regex's, channel type resolutions and other common functions that would be really nice to get from 1 centralized place. That's where @sapphire/discord.js-utilities come in. This package provides common Discord specific utilities for your Discord bots.
```
Comment (Favna) [inline packages/discord.js-utilities/README.md:21]:
```suggestion
There many functions that are always extremely useful to have when developing bots with [Discord.JS]. Such as channel type guards, paged messages and more. For this purpose we provide @sapphire/discord.js-utilities. This is also an extension of sorts on our package [@sapphire/discord-utilities], which is agnostic to the API library you're using for your bot. That is to say, @sapphire/discord.js-utilities re-exports all functions from [@sapphire/discord-utilities]
```

And add to the bottom of the readme:
```md
[Discord.JS]: https://discord.js.org
[@sapphire/discord-utilities]: ../discord-utilities
```

(gotta check if that last one works but can only check once you're committed it to your branch)

File: packages/discord.js-utilities/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@sapphire/discord.js-utilities",
+	"version": "1.0.0",
```
Comment (Favna) [inline packages/discord.js-utilities/package.json:3]:
```suggestion
	"version": "0.0.1",
```

First major release will be set by Lerna

File: packages/discord.js-utilities/package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		"type": "git",
+		"url": "git+https://github.com/sapphire-project/utilities.git",
+		"directory": "packages/discord.js-utilities"
+	},
+	"files": [
+		"dist",
+		"!dist/*.tsbuildinfo"
+	],
+	"engines": {
+		"node": ">=12",
```
Comment (Favna) [inline packages/discord.js-utilities/package.json:33]:
```suggestion
		"node": ">=14",
```

File: packages/discord.js-utilities/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export * from '@sapphire/discord-utilities';
+export * from './lib/type-guards';
```
Comment (Favna) [inline packages/discord.js-utilities/src/index.ts:2]: You forgot to export your own PaginatedMessage stuff here

**Review by Favna — CHANGES_REQUESTED** [review]
Leaving review of PaginatedMessage and LazyPaginatedMessage up to Kyra/Vlad, here's some stuff already.

File: packages/discord.js-utilities/src/index.ts (hunk 4 above)
Comment (RealShadowNova) [reply packages/discord.js-utilities/src/index.ts:2]: Yeah, just now seeing that, thanks.

**Review by Favna — APPROVED** [review]

**Pull request merged**